packages/sdk/src/utils/listener.ts

```diff
@@ -78,7 +78,7 @@ export class TransactionManagerListener {
         receivingChainId: txData.receivingChainId.toNumber(),
         callData: txData.callData,
         transactionId: txData.transactionId,
-        blockNumber: txData.blockNumber.toNumber(),
+        preparedBlockNumber: txData.blockNumber.toNumber(),
```

Comment:
We will need to reflect this change in the subgraph once we redeploy. Shouldn't block merging, just a reminder

Reply:
@rhlsthrm took care of this.